pkg/stanza/fileconsumer/attrs/attrs.go

```
@@ -32,6 +39,28 @@ func (r *Resolver) Resolve(path string) (attributes map[string]any, err error) {
 	if r.IncludeFilePath {
 		attributes[LogFilePath] = path
 	}
+	if r.IncludeFileOwner || r.IncludeFileGroup {
+		var file, fileErr = os.OpenFile(fmt.Sprint(path), os.O_RDONLY, 0000)
+		if fileErr == nil {
```

We should return the error rather than ignore it.